test_runner/src/test/kotlin/ftl/util/ProgressBarTest.kt

```diff
@@ -49,11 +49,12 @@ class ProgressBarTest {
         runWithProgress(
             startMessage = "TEST",
             action = { action.run() },
+            onSuccess = {},
```

Comment:
We dont need teh onSuccess as it has a default?

Reply:
fixed